pex/variables.py

```diff
@@ -21,15 +25,17 @@ class Variables(object):
 
     @classmethod
     def process_pydoc(cls, pydoc):
+        # type: (Optional[str]) -> Tuple[str, str]
         if pydoc is None:
```

Comment:
Possibly consider extending this to extract type hints in the future.

Reply:
Once we require Python 3, that would be super easy to do with `typing.get_type_hints()`, which is how the Pants rule signature parsing code works.